pkg/util/finalizer/patch_finalizer.go

```diff
@@ -0,0 +1,18 @@
+package finalizer
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Patch(object metav1.Object, prevFinalizer, finalizer string) {
```

Comment:
Is this function used anywhere?

Reply:
nope, it was supposed to and got dismissed (sad because I have tests for it too)